numpy/core/tests/test_function_base.py

```diff
@@ -185,18 +198,34 @@ def test_start_stop_array_scalar(self):
 
     def test_start_stop_array(self):
         # Try to use all special cases.
-        start = array([1.e0, 32., 1j, -4j, 1+1j, -1])
-        stop = array([1.e4, 2., 16j, -324j, 10000+10000j, 1])
+        start = array([1.e0, 32., 1j, -4j, 1+1j, -1, 0.3])
+        stop = array([1.e4, 2., 16j, -324j, 10000+10000j, 1, 20.3])
+
         t1 = geomspace(start, stop, 5)
         t2 = stack([geomspace(_start, _stop, 5)
                     for _start, _stop in zip(start, stop)], axis=1)
         assert_equal(t1, t2)
+        for i in range(len(start)):
+            assert_equal(t1[0, i], start[i])
+            assert_equal(t1[-1, i], stop[i])
+            assert_equal(t2[0, i], start[i])
+            assert_equal(t2[-1, i], stop[i])
```

Comment:
```suggestion
```
This test isn't interesting, it's purpose is just to check that the array version behaves like the non-array.

Reply:
Same for all of the changes in this function

Reply:
Correct - will revert.